RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -900,65 +885,121 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
+
+    Float_t eventTime[10];
+    Float_t eventTimeRes[10];
+    Double_t eventTimeWeight[10];
+
+    for (Int_t i = 0; i < TOFResponse.GetNmomBins(); i++)
+    {
+      if (i >= 10)
+        AliFatal("Index is too high!");
+      Float_t mom = (TOFResponse.GetMinMom(i) + TOFResponse.GetMaxMom(i)) / 2.f;
+      eventTime[i] = TOFResponse.GetStartTime(mom) * 1.e-3;       // ps to ns
+      eventTimeRes[i] = TOFResponse.GetStartTimeRes(mom) * 1.e-3; // ps to ns
+      eventTimeWeight[i] = 1. / (eventTimeRes[i] * eventTimeRes[i]);
+
+      //PH The part below is just a place holder
+      if (TOFResponse.GetStartTimeMask(mom) & 0x1)
+        SETBIT(collision.fCollisionTimeMask, 0);
+      else
+        CLRBIT(collision.fCollisionTimeMask, 0);
+      //
+      if (TOFResponse.GetStartTimeMask(mom) & 0x2)
+        SETBIT(collision.fCollisionTimeMask, 1);
+      else
+        CLRBIT(collision.fCollisionTimeMask, 1);
+      //
+      if (TOFResponse.GetStartTimeMask(mom) & 0x3)
+        SETBIT(collision.fCollisionTimeMask, 2);
+      else
+        CLRBIT(collision.fCollisionTimeMask, 2);
+    }
 
-  // Recalculate unique event time and its resolution
-  collision.fCollisionTime = AliMathBase::TruncateFloatFraction(TMath::Mean(10, eventTime, eventTimeWeight), mCollisionPosition);                 // Weighted mean of times per momentum interval
-  collision.fCollisionTimeRes = AliMathBase::TruncateFloatFraction(TMath::Sqrt(9. / 10.) * TMath::Mean(10, eventTimeRes), mCollisionPositionCov); // PH bad approximation
+    // Recalculate unique event time and its resolution
+    collision.fCollisionTime = AliMathBase::TruncateFloatFraction(TMath::Mean(10, eventTime, eventTimeWeight), mCollisionPosition);                 // Weighted mean of times per momentum interval
+    collision.fCollisionTimeRes = AliMathBase::TruncateFloatFraction(TMath::Sqrt(9. / 10.) * TMath::Mean(10, eventTimeRes), mCollisionPositionCov); // PH bad approximation
+
+    FillTree(kEvents);
+
+    //---------------------------------------------------------------------------
+    // Run 2 Collision information
+    run2collinfo.fVertexType = vertexType;
+    run2collinfo.fEventCuts = 0; //No event selection passed, now check which passed
+    
+    // Get multiplicity selection
+    AliMultSelection *multSelection = (AliMultSelection*) fESD->FindListObject("MultSelection");
+    if (!multSelection)
+      AliFatal("MultSelection not found in input event");
+    if( multSelection->GetThisEventINELgtZERO() )
+      SETBIT (run2collinfo.fEventCuts, kINELgtZERO);
+    
+    if( multSelection->GetThisEventIsNotPileupInMultBins() )
+      SETBIT (run2collinfo.fEventCuts, kPileupInMultBins);
+    
+    if( multSelection->GetThisEventHasNoInconsistentVertices() )
+      SETBIT (run2collinfo.fEventCuts, kConsistencySPDandTrackVertices);
+    
+    if( multSelection->GetThisEventPassesTrackletVsCluster() )
+      SETBIT (run2collinfo.fEventCuts, kINELgtZERO);
```

Comment:
Shouldn't this be a different bit?

Reply:
Yes, I noticed it about 15 minutes before you wrote here :-) 
Thanks!